model/config.go

```diff
@@ -385,6 +385,7 @@ type ServiceSettings struct {
 	EnableCustomGroups                                *bool   `access:"site_users_and_teams"`
 	SelfHostedFirstTimePurchase                       *bool   `access:"write_restrictable,cloud_restrictable"`
 	AllowSyncedDrafts                                 *bool   `access:"site_posts"`
+	ExperimentalMaxUserPreferences                    *int    `access:"experimental_features,site_users_and_teams"`
```

Comment:
Not sure about these tags...

Reply:
If you don't want the setting accessible via the system console UI then I think there's no need to make it accessible via the API and therefore you can omit the `access` tag.

Here's some optional more detail about the `access` tag:

The `access` tag authorizes access to that config setting field with any one of the permissions listed. 

Each of the values have an associated read (`sysconsole_read_...`) and write (`sysconsole_write_...`) permission. For example, `access:"site_users_and_teams"` restricts reading the field's value to users with the permission `sysconsole_read_site_users_and_teams` and for writing by users with the permission `sysconsole_write_site_users_and_teams`. The `sysconsole_...` permissions are distributed to roles that are configured to provided a subset of system administration permissions.

There's a hard-to-find Godoc about the `access` tag on `model.Config`: https://github.com/mattermost/mattermost-server/blob/06e964b86b0d4a27f2efdf70d2c5232ec3353625/model/config.go#L3094-L3128.

Reply:
Thanks for the clarification! I've removed them for now as this is not the intended use for this config.